### PR TITLE
Ticket sales and grants for Atlantic 2023

### PR DESCRIPTION
**Pull request by mxsasha**
- [x] Finalise blog post
- [x] Tito to live
- [x] Actually set up grant program
- [x] Update blog post date
- [x] ~~Figure out which mailing lists this go to and Eric to make that happen~~ This one Atlantic only, as talks are announced quite soon and we should do that wider.

Blog post: https://writethedocs-www--1964.org.readthedocs.build/conf/atlantic/2023/news/tickets-grants/
Note that links in the blog post link to the live website and may therefore still be outdated.

----
:books: Documentation preview :books:: https://writethedocs-www--1964.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2023/news/tickets-grants.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Greetings, documentarians! Today we're happy to announce that tickets are `on sale now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_ and our `Opportunity Grant program <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ is open.
+
+Following the success of our previous virtual conferences, we look forward to our first Atlantic conference.
+The Call for Papers has closed and we're currently reviewing all proposals. We're looking forward to announcing
+our speakers soon.
+
+
+Get your ticket for the conference
+----------------------------------
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/tickets-grants.rst:17]: Let's remind folks about the timezone?

Comment (plaindocs) [reply docs/conf/atlantic/2023/news/tickets-grants.rst:17]: And say we'll be announcing talks next week or some such

File: docs/conf/atlantic/2023/news/tickets-grants.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
+:template: {{year}}/generic.html
+
+.. post:: May 1st, 2023
+   :tags: {{shortcode}}-{{year}}, website, tickets
+
+Announcing Ticket Sales and Grant Program
+=========================================
+
+Greetings, documentarians! Today we're happy to announce that tickets are `on sale now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_ and our `Opportunity Grant program <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ is open.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/news/tickets-grants.rst]:
```suggestion
Greetings, documentarians! Today we're happy to announce that tickets for {{name}} conference are `on sale now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_ and our `Opportunity Grant program <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_ is open.
```

**Review by plaindocs — APPROVED** [review]
LGTM

**Review by PeaceOloruntoba — COMMENTED** [review]
wow!

**Pull request merged**